exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,10 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+
+let percentageOfStudents = ((15 / 23) *100)
+//console.log(percentageOfTrainees)
```

Comment:
You shouldn't leave commented out code in a submission. If you put something into the code for debugging, and you don't need it any more, delete it.